chain/gen/gen.go

```diff
@@ -529,24 +540,33 @@ func IsRoundWinner(ctx context.Context, ts *types.TipSet, round int64, miner add
 		return false, nil, nil
 	}
 
-	proof, err := epp.ComputeProof(ctx, sectors, hvrf[:], winners)
+	return true, &ProofInput{
```

Comment:
can drop the boolean return value, and use the proofinput being non-nil for the same check